app/uk/gov/hmrc/estates/controllers/transformers/register/DeceasedTransformationController.scala

```diff
@@ -39,10 +40,10 @@ class DeceasedTransformationController @Inject()(
                                                 )(implicit val executionContext: ExecutionContext)
   extends EstatesBaseController(cc) {
 
-  private val logger = LoggerFactory.getLogger("application." + this.getClass.getCanonicalName)
+  private val logger: Logger = Logger(getClass)
 
   def get: Action[AnyContent] = identify.async {
-    request =>
+    implicit request =>
```

Comment:
Why implicit added?

Reply:
I needed to add it to the save method for the session ID and having it implicit is the convention elsewhere so I added it to all methods on the controller. Happy to remove it for the ones it's not currently necessary for though?

Reply:
No, it's ok.